speechAssets/IntentSchema.json

```diff
@@ -13,6 +13,15 @@
       "intent": "GetServiceAdvisories"
     },
     {
+      "intent": "SetHomeStation",
+      "slots": [
+        {
+          "name": "homeStation",
```

Comment:
True. I think I named the slots different things for different intents because it was like that in a tutorial, but I see in some of the sample skills they are naming them the same, so it must be fine to do it that way.